src/facet_display/MultiFacetGroup.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react"
+
+import { SearchFacetItem } from "./SearchFacetItem"
+import type {
+  Bucket,
+  MultiFacetGroupOptions,
+  Facets,
+  BooleanFacets
+} from "./types"
+
+interface Props {
+  results: Record<string, Bucket[]>
+  facets: MultiFacetGroupOptions["facets"]
+  onUpdate: React.ChangeEventHandler<HTMLInputElement>
+  activeFacets: Facets & BooleanFacets
+}
+
+// NOTE the manual type predicate should be unncessary in TS >= 5.5
+const hasBucket = <T extends { bucket?: Bucket }>(
+  f: T
+): f is T & { bucket: Bucket } => !!f.bucket
+
+/**
+ * Used to display a specific set of facet name-value pairs.
+ */
+function MultiFacetGroup(props: Props) {
```

Comment:
This seems a bit weird to me since functionally each of the facets are independent. It seems bad that if we wanted to make the different boolean facets into separate blocks we'd need multiple  MultiFacetGroup components with one facet each or a new component

Reply:
We confirmed with @Ferdi that having multi facets grouped together is the desired behavior, in which case I think the above approach makes sense.